src/tasks/task2.js

```diff
@@ -0,0 +1,72 @@
+import fs from 'fs';
+import csv from 'csvtojson';
+import * as logger from '../../services/logger';
+
+const _filePath = './scv/node_mentoring_t1_2_input_example.csv';
+const _reader = fs.createReadStream(_filePath);
+const _writer = fs.createWriteStream('./task2.txt');
+
+const _convertoToJson = chunk => {
+  return csv()
+    .fromString(chunk)
```

Comment:
How about using .fromStream() from csv library to asynchronously process line by line?